store/auth.go

```diff
@@ -200,9 +200,9 @@ func RequestPackageAccessMacaroon() (string, error) {
 	return responseData.Macaroon, nil
 }
 
-// RequestDischargeMacaroon requests a discharge macaroon for an existing macaroon
-func RequestDischargeMacaroon(username, password, macaroon, otp string) (string, error) {
-	const errorPrefix = "cannot get discharge macaroon: "
+// DischargeAuthCaveat returns a macaroon with the store auth caveat discharged
```

Comment:
I reckon this was a self-review, btw, since the code was mine. The style hint is likely useful nevertheless, though.